Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -187,7 +187,14 @@ class BackForwardListViewController: UIViewController, UITableViewDataSource, UI
             if snappedToBottom {
                 verticalConstraints += [make.bottom.equalTo(self.view).offset(-bvc.footer.frame.height).constraint]
             } else {
-                verticalConstraints += [make.top.equalTo(self.view).offset(bvc.header.frame.height + UIApplication.shared.statusBarFrame.size.height).constraint]
+                var statusBarHeight: CGFloat = 0
+                if #available(iOS 13.0, *) {
+                    let window = UIApplication.shared.windows.filter {$0.isKeyWindow}.first
```

Comment:
I also have the impression here that using `UIWindow.keyWindow` would work here? This would avoid us that check for `if #available(iOS 13.0, *) {`

Reply:
yes indeed, I didn't realize in the other PR 😄 thanks!